packages/test-framework/src/react-testing-library-support/CheckoutPageNodeObject.ts

```diff
@@ -87,81 +91,81 @@ export class CheckoutPageNodeObject {
         this.server.use(handler);
     }
 
-    use(preset: string): void {
+    use(preset: CheckoutPreset): void {
         switch (preset) {
-            case 'CartWithBillingEmail':
+            case CheckoutPreset.CheckoutWithBillingEmail:
                 this.server.use(
                     rest.get('/api/storefront/checkout/*', (_, res, ctx) =>
-                        res(ctx.json(cartWithBillingEmail)),
+                        res(ctx.json(checkoutReadyForShippingStepWithGuestEmail)),
```

Comment:
Maybe we can call it `checkoutWithBillingEmail` since the above is too verbose and hard to remember, eventually I feel folks will create duplicate as the name is not straightforward to think of/remember.